### PR TITLE
Use mesh_coloration from kwiver.

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@mleotta These are corresponding changes in Telesculptor.

File: gui/ColorizeSurfaceOptions.h [hunk 1]
```diff
@@ -42,6 +42,10 @@ class vtkActor;
 
 class ColorizeSurfaceOptionsPrivate;
 class MeshColoration;
+namespace kwiver { namespace arrows { namespace vtk {
```
Comment (mleotta) [inline gui/ColorizeSurfaceOptions.h:45]: I would prefer each namespace start on a new line

File: gui/MainWindow.h [hunk 2]
```diff
@@ -43,7 +43,9 @@
 class CameraView;
 class ToolData;
 class WorldView;
+namespace kwiver { namespace arrows { namespace vtk {
```
Comment (mleotta) [inline gui/MainWindow.h:46]: start namespaces on new lines

File: gui/WorldView.h [hunk 3]
```diff
@@ -47,13 +47,16 @@ class vtkObject;
 class vtkPolyData;
 class vtkStructuredGrid;
 class MeshColoration;
+namespace kwiver { namespace arrows { namespace vtk {
```
Comment (mleotta) [inline gui/WorldView.h:50]: namespaces on new lines

**Review by mleotta — CHANGES_REQUESTED** [review]
I made a few very minor formatting suggestions.  Generally this looks good, but it will need to wait until the KWIVER branch lands.

**Comment by danlipsa** [comment]
@mleotta TeleSculptor now picks up kwiver master and has changes required for mesh_coloration leaving in kwiver.

**Comment by mleotta** [comment]
@danlipsa this is failing CI because of a recent change in KWIVER that now requires CMake >= 3.15.  Travis CI uses Ubuntu 18.04, which comes with CMake 3.10.  I've updated the Travis CI scripts to download a newer CMake.

Can you merge in the latest TeleSculptor master and push this again to see if that gets CI working again on this branch?

**Comment by danlipsa** [comment]
@mleotta I rebased on master so lets see now.

**Pull request merged**